hive/Dockerfile

```diff
@@ -30,8 +30,4 @@ ADD hive_sql.hql $HIVE_HOME/
 
 WORKDIR $HIVE_HOME
 
-ADD bootstrap.sh /etc/bootstrap.sh
-RUN chmod +x /etc/bootstrap.sh
-
-
-CMD ["/etc/bootstrap.sh", "-d"]
\ No newline at end of file
+CMD ["/bin/bash"]
```

Comment:
this change is strange as your container does not start hive at all

Reply:
Hive is started by the task and need some config files to be instanciated by the task before starting. Therefore, we cannot start hive at the docker run level via a bootstrap script. And i decided to follow the same logic as for hdfs-spark.

Reply:
ok